src/Analyzers/Core/CodeFixes/MatchFolderAndNamespace/AbstractChangeNamespaceToMatchFolderCodeFixProvider.cs

```diff
@@ -29,15 +33,21 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
                 context.RegisterCodeFix(
                     CodeAction.Create(
                         AnalyzersResources.Change_namespace_to_match_folder_structure,
-                        cancellationToken => FixAllInDocumentAsync(context.Document, context.Diagnostics, cancellationToken),
+                        cancellationToken => FixAllInDocumentAsync(context.Document, context.Diagnostics,
+#if CODE_STYLE
```

Comment:
so, practically, i thought we end up running the CODE_STYLE analyzers in roslyn itself, because we defer to those if available.  Does this mean we're protentially regressing user style options?  How does this end up practically falling out for users @jmarolf ?

Reply:
Good question. The options that are passed here (or not in code style layer) are the default values of options stored in the IDE registry keys. Code style layer does not have access to these values currently.